documentation/release-notes/topics/version_4.rst

```diff
@@ -2,12 +2,90 @@
+
+=======================================
+Enhanced Licensed Feature: Connectivity
+=======================================
+- Connections and Circuits have been enhanced to allow users to attach documents, images, and links.
+
+- You can also now add Work Notes to Connections and Circuits.
+
+========================================
+Enhanced Feature: Alarm Event Management
+========================================
+- A new *Assets -> Events* page was added to the application to allow for consolidated events management. The events displayed will be a global view of all events on assets the user can access.
+
+- Users can export, filter, and sort the events list by various criteria.
+
+- Events can be acknowledged or closed individually and in bulk.
+
+=================================================
+Enhanced Feature: Notifications -> Alarm Policies
+=================================================
+- The notification template has been updated to aggregate multiple events in one email. This enhancement will reduce email noise in the case of event spikes.
+
+- Administrators can now select All or multiple asset types from the same Alarm Policy. Previously, users were allowed to choose a single asset type for policy.
+
+- Administrators can now select a notification channel for an Alarm Policy; more information on Notification Channels is below.
+
+========================================================
+Enhanced Feature: Notifications -> Notification Channels
+========================================================
+- This feature allows Administrators to create an Alarm Policy to channel notifications to an external communication or alerting system such as Microsoft Teams.
+
+- This release adds the integration with Microsoft Teams. Administrators can configure a link to a specific Microsoft Teams Channel. Administrators can add multiple channels and target them with different Alarm Policies.
+
+================================================
+Enhanced Feature: Linux & Windows Data Collector
+================================================
+- VMware protocol has been enhanced to add a monitoring pipeline for discovered sensors. To use this enhancement, customers must update to the latest version of the Data Collector and rediscover the assets.
+
+======================================
+Other notable changes and improvements
+======================================
+- Data grids will save column selection and sorting order by default.
+
+- Launch Web Interface has been added as a primary action button on device asset types.
+
+- Volume Unit has been added as a :ref:`locale setting <Locale-settings-doc>` in the application.
+
+- The Documents section has been moved to be a main navigation menu item. Previously, it was under the Assets section.
+
+- The location layout editor now supports adding triangular shapes.
+
+- Sorting order has been updated to be case-insensitive alphanumeric for devices that report multiple serial numbers.
```

Comment:
We also consistently order other things in search results that have more than one thing.  Examples include MAC addresses, IP Addresses, NIC descriptions, etc

Reply:
Fixed.